README.md

```diff
@@ -1,13 +1,78 @@
-# pa-anticipatory-action
+# OCHA Anticipatory Action Framework
+**Note: the code presented in this repository is work in progress. If you have questions or run into any issues, please contact us at centrehumdata@un.org.**
+
+An OCHA anticipatory action framework (AAF) comprises three pre-agreed pillars: 
+financing, activities, and a trigger. 
+The trigger defines an exceptional shock (cause and impact) to which the framework should respond. 
+The framework activates when projections indicate that the shock is likely to realise. 
+Funds are then rapidly disbursed and interventions are implemented. 
+
+The trigger is based on data and enables the automated activation of the framework. 
+This repository contains the triggers that are part of the several AAF pilots. 
+While the triggers are adjusted to the local context, most of the data processing is transferable to other countries.
+
+The AAF pilots implemented as of January 2021 are:
```

Comment:
Are there any completed trigger cards that we could link to (the templates that Josee created)?

Reply:
@joseepoirier any completed trigger cards we could link to? 😄 Or other public documents (for Bangladesh maybe the blog?)?

Reply:
Now included link to Bangladesh blogpost. @joseepoirier let us know if there is any other public material that would be good to link to :) Merging for now.